presto-verifier/src/test/java/com/facebook/presto/verifier/framework/TestCreateViewVerification.java

```diff
@@ -141,13 +141,14 @@ public void testFailed()
     @Test
     public void testRunningInQueryBankMode()
     {
-        String query = "CREATE VIEW succeeded_not_exists AS SELECT * FROM tpch.tiny.nation";
+        String snapshotModeQuery = "CREATE VIEW succeeded_not_exists_snapshot AS SELECT * FROM tpch.tiny.nation";
 
-        Optional<VerifierQueryEvent> event = runVerification(query, query, saveSnapshotSettings);
+        Optional<VerifierQueryEvent> event = runVerification(snapshotModeQuery, snapshotModeQuery, saveSnapshotSettings);
         assertTrue(event.isPresent());
         assertEvent(event.get(), SUCCEEDED);
 
-        event = runVerification(query, query, queryBankModeSettings);
+        String queryBankModeQuery = "CREATE VIEW succeeded_not_exists_querybank AS SELECT * FROM tpch.tiny.nation";
+        event = runVerification(queryBankModeQuery, queryBankModeQuery, queryBankModeSettings);
```

Comment:
Should we drop these views after tests too?

Reply:
i looked at this more, and I don't think we should. The verifier both creates and tears down these views and should do it consistently.   It also adds custom prefixes to the view names before creating the view.  Dropping the views could theoretically fail, but it's not more likely to fail than if we did it ourselves.